apps/bfd-server/bfd-server-war/src/main/java/gov/cms/bfd/server/war/stu3/providers/TransformerUtils.java

```diff
@@ -3369,12 +3369,24 @@ public static Bundle addResourcesToBundle(Bundle bundle, List<IBaseResource> res
     return bundle;
   }
 
+  /**
+   * @param beneIds the {@link Collection} of beneficiary IDs top log {@link
+   */
   public static void logBeneIdToMdc(Collection<String> beneIds) {
     if (!beneIds.isEmpty()) {
       MDC.put("bene_id", String.join(", ", beneIds));
     }
   }
 
+  /**
+   * @param beneId the {@link Long} beneficiary ID to log {@link
+   */
+  public static void logBeneIdToMdc(Long beneId) {
+    if (beneId != null) {
+      MDC.put("bene_id", String.valueOf(beneId));
+    }
+  }
+
```

Comment:
same comment as above. and actually this is a shame that we have all of this twice but we are doing some work early next sprint to fix some other issues with this bene_id logging so we will sort that out then so I don't think it should become scope creep on this PR.